homeassistant/helpers/helper_config_entry_flow.py

```diff
@@ -337,3 +366,22 @@ def wrapped_entity_config_entry_title(
     if state:
         return state.name or object_id
     return object_id
+
+
+@callback
+def exclude_own_entities(
+    handler: HelperConfigFlowHandler | HelperOptionsFlowHandler,
+    config: dict[str, Any],
+) -> vol.Schema:
+    """Return an entity selector which excludes own entities."""
+    if not isinstance(handler, HelperOptionsFlowHandler):
+        return selector.selector({"entity": config})
+
+    entity_registry = er.async_get(handler.hass)
+    entities = er.async_entries_for_config_entry(
+        entity_registry,
+        handler._config_entry.entry_id,  # pylint: disable=protected-access
```

Comment:
Should it become `handler.config_entry` since it's now used in this method outside of the class

Reply:
Fixed in cc8a589